user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/AuditErrorMessage.java

```diff
@@ -54,13 +54,13 @@ public Throwable getError() {
     }
 
     /**
-     * Builder class for the creation of instances of the AuditSuccessMessage.  This is a variant of the Fluent Builder
+     * Builder class for the creation of instances of the AuditErrorMessage. This is a variant of the Fluent Builder
      * which will use Java Objects or JsonNodes equivalents for the components in the build.
```

Comment:
We don't use JsonNodes in the building of audit messages.  Seen a few of these in a module that I'm looking at.  Suggest you remove this and check the rest of the audit message for the same.